deployments/run.sh

```diff
@@ -52,7 +52,7 @@ COMPOSE_ARGS+=$BRIDGE_PATH/.env
 grep -e MATRIX_ACCESS_TOKEN -e WITH_PROXY $BRIDGE_PATH/.env > .env2 && . ./.env2 && rm .env2
 
 if [ ! -z ${MATRIX_ACCESS_TOKEN+x} ]; then
-	sed -i '' -e "s/access_token.*/access_token: \"$MATRIX_ACCESS_TOKEN\"/" ./monitoring/grafana-matrix/config.yml
+	sed -e "s/access_token.*/access_token: \"$MATRIX_ACCESS_TOKEN\"/" ./monitoring/grafana-matrix/config.yml
```

Comment:
Does this really work without `-i` ? I thought that without `--in-place` it would simply output the changes instead of applying them?

Reply:
You're right, I mixed up the flags. It should be using `-i` instead of `-e`